src/AcceptanceTests/AcceptanceTests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
Should this be called out in upgrade guide? Similar to what we had to do to ASB [here](https://docs.particular.net/transports/upgrades/asb-8to9#net-framework-4-6-required) here? We also concluded that was a major version because we can't "assume" people will have the new framework and therefore it would not qualify as a minor release?